keras_cv/src/layers/preprocessing/jittered_resize.py

```diff
@@ -103,6 +103,7 @@ def __init__(
         crop_size=None,
         bounding_box_format=None,
         interpolation="bilinear",
+        minimum_box_area_ratio=0.0,
```

Comment:
let us remove this from the augmentation layers. We need to keep this consistent with the other layers. Let us not touch this.